src/orion/core/worker/storage_backup.py

```diff
@@ -0,0 +1,473 @@
+
+    src_storage: BaseStorageProtocol = setup_storage(
+        {"database": {"host": load_host, "type": "pickleddb"}}
+    )
+    logger.info(f"Loaded src {load_host}")
+
+    import_benchmarks = False
+    _describe_import_progress(STEP_COLLECT_EXPERIMENTS, 0, 1, progress_callback)
+    if name is None:
+        import_benchmarks = True
+        # Retrieve all src experiments for export
+        experiments = src_storage.fetch_experiments({})
+    else:
+        # Find experiments based on given name and version
+        query = {"name": name}
+        if version is not None:
+            query["version"] = version
+        experiments = src_storage.fetch_experiments(query)
+        if not experiments:
+            raise DatabaseError(
+                f"No experiment found with query {query}. Nothing to import."
+            )
+        if len(experiments) > 1:
+            experiments = sorted(experiments, key=lambda d: d["version"])[:1]
+        logger.info(
+            f"Found experiment {experiments[0]['name']}.{experiments[0]['version']}"
+        )
+    _describe_import_progress(STEP_COLLECT_EXPERIMENTS, 1, 1, progress_callback)
+
+    preparation = _prepare_import(
+        src_storage,
+        storage,
+        resolve,
+        experiments,
+        import_benchmarks,
+        progress_callback=progress_callback,
+    )
+    _execute_import(storage, *preparation, progress_callback=progress_callback)
+
+
+def _dump(src_storage, dst_storage, name=None, version=None):
+    """Dump data from source storage to destination storage.
+
+    Parameters
+    ----------
+    src_storage: BaseStorageProtocol
+        input storage
+    dst_storage: BaseStorageProtocol
+        output storage
+    name:
+        (optional) if provided, dump only data related to experiment with this name
+    version:
+        (optional) version of experiment to dump
+    """
+    # Get collection names in a set
+    if name is None:
+        # Nothing to filter, dump everything
+        # Dump benchmarks
+        logger.info("Dumping benchmarks")
+        for benchmark in src_storage.fetch_benchmark({}):
+            dst_storage.create_benchmark(benchmark)
+        # Dump experiments
+        logger.info("Dumping experiments, algos and trials")
+        for i, src_exp in enumerate(src_storage.fetch_experiments({})):
+            logger.info(f"Dumping experiment {i + 1}")
+            _dump_experiment(src_storage, dst_storage, src_exp)
+    else:
+        # Get experiments with given name
+        query = {"name": name}
+        if version is not None:
+            query["version"] = version
+        experiments = src_storage.fetch_experiments(query)
+        if not experiments:
+            raise DatabaseError(
+                f"No experiment found with query {query}. Nothing to dump."
+            )
+        if len(experiments) > 1:
+            exp_data = sorted(experiments, key=lambda d: d["version"])[0]
+        else:
+            (exp_data,) = experiments
```

Comment:
```suggestion
exp_data = sorted(experiments, key=lambda d: d["version"])[0]
```
Simpler IMHO than a non-significant optimization for n==1.

Reply:
Ok !